go.mod

```diff
@@ -20,11 +20,11 @@ require (
 	github.com/google/gopacket v1.1.19
 	github.com/google/uuid v1.6.0
 	github.com/gorilla/mux v1.8.1
-	github.com/gorilla/websocket v1.5.1
+	github.com/gorilla/websocket v1.5.3
```

Comment:
@simondeziel ah sorry I missed this, but we need to keep websocket pinned at 1.5.1 for all branches currently as 1.5.3 was problematic on riscv64 (and is a pre-release anyway according to their github page).